tests/testthat/test-weighting.R

```diff
@@ -124,5 +152,12 @@ with_test_authentication({
                 array(c(110, 100), dim=2L, dimnames=list(v4=c("B", "C"))))
         })
     })
-
+with(test.dataset(df), {
+        test_that("makeWeight returns the expected weights", {
+            ds$weight <- makeWeight(ds$v4 ~ c(30, 70, 0), name = "weight")
+            expected_weights <- c(0.6, 1.4, 0.6, 1.4, 0.6, 1.4, 0.6, 1.4, 0.6, 1.4, 0.6, 1.4,
+                                  0.6, 1.4, 0.6, 1.4, 0.6, 1.4, 0.6, 1.4)
```

Comment:
This is surely right, but I'm surprised that these are the weights given that everything is divided by 100 at L151 in weights.R